polaris-react/stylelint/plugins/coverage/configs/index.js

```diff
@@ -0,0 +1,127 @@
+module.exports = {
+  plugins: ['stylelint-scss', '../'],
+  rules: {
+    'at-rule-disallowed-list': ['keyframes'],
+    'color-named': 'never',
+    'color-no-hex': true,
+    'declaration-property-value-disallowed-list': [
+      {
+        '/color$/': ['/#[0-9a-f]{3,6}/'],
+        display: ['grid', 'flex'],
+        'font-weight': ['/[0-9]*/'],
+        opacity: ['/?![01]'],
```

Comment:
Are we trying to error on anything that is not `0`, `1`, or a custom property?

If so, I believe this is what we want: [/^(?!0|1)\d+/](https://regex101.com/r/nGthny/1)

Reply:
Yeah you got it!